akka-stream/src/main/boilerplate/akka/stream/scaladsl/GraphApply.scala.template

```diff
@@ -26,7 +26,7 @@ trait GraphApply {
    */
   def create[S <: Shape, Mat](g1: Graph[Shape, Mat])(buildBlock: GraphDSL.Builder[Mat] ⇒ (g1.Shape) ⇒ S): Graph[S, Mat] = {
     val builder = new GraphDSL.Builder
-    val s1 = builder.add(g1)
+    val s1 = builder.add(g1, Keep.right)
```

Comment:
We import `g1`, hence we need its materialized value while the current module is `EmptyModule`, so there is no materialized value to keep from it: meaning `Keep.right` is needed.